src/porepy/models/momentum_balance.py

```diff
@@ -865,6 +865,7 @@ class MomentumBalance(  # type: ignore[misc]
     ConstitutiveLawsMomentumBalance,
     BoundaryConditionsMomentumBalance,
     SolutionStrategyMomentumBalance,
+    pp.FluidMixin,
```

Comment:
The mechanics-only model is a unique in the sense, that it doesn't need as you said.
But since the respective solution strategy inherits from the base `SolutionStrategy`, the base class would require special logic to treat this unique case.

I then realized, that the mechanics-only model so far also had a redundant `fluid: FluidConstants` (probably for the same reasons). So I decided to go with that and now it has a redundant `fluid: Fluid` attribute. But since that one requires some this mixin, this redundancy became more visible now.

Reply:
I see. What part of the SolutionStrategy requires the fluid, specifically?

Reply:
The `prepare_simulation() -> set_materials()` call-stack, where `solid` and `fluid` are created.
Here we would need some, as I see it, ugly code checking base classes and inheritance, if we want a conditional creation of the fluid.
Ugly, because I cannot foresee how many multi-physics model will come in addition to the existing ones, and how to programmatically determine that only `MomentumBalance` is used, without altering the code every time a new model is introduced inheriting from `MomentumBalance`.

Reply:
I see. I can cook up an alternative solution. But I don't think it has any significant benefits, so let's leave as is.

Reply:
I added a short comment in the code.